platform/viewer/src/hooks/useHotkeys.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * @param {Object} manager HotkeysManager instance
+ * @param {Object} hotkeys hotkey bindings
+ * @param {Object} defaultHotkeys default hotkey bindings
+ */
+const useHotkeys = (manager, hotkeys, defaultHotkeys) => {
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+    if (!hotkeys) {
+      console.warn('[hotkeys] No bindings defined for hotkeys hook!');
+      return;
+    }
+
+    console.debug('[hotkeys] Setting up hotkeys...');
+    manager.setDefaultHotKeys(defaultHotkeys || hotkeys);
+    manager.setHotkeys(hotkeys);
+    setIsLoaded(true);
+
+    return () => {
+      console.debug('[hotkeys] Removing hotkeys...');
+      manager.destroy();
+    };
+  }, [manager, hotkeys, defaultHotkeys]);
```

Comment:
I don't know that we need react to monitor the manager or the mode config. If someone wants to change these, they would probably interact with the hotkeysManager directly? Or we'll have somewhere in the pipe where userPreferences influence this?

Then on destroy, this will clean up whatever the active/current values are in the manager, regardless of how they've changes or current state.